### DISCUSSION TITLE
Syntax validation feature for python mode

### DISCUSSION THREAD

**Discussion by prakalathan** [ajaxorg/ace#4914] [Ideas]
**Does ace-builds supports syntax check/validate for python mode/language like javascript/html/php doing?**
![javascript-supported syntax check](https://user-images.githubusercontent.com/25406424/188083682-2a9a73bb-d958-4848-a8bc-a52fc0e40132.png)

### For 'Javascript' mode which is working for syntax checking as well. But for 'Python' mode we didn't check the syntax as validated.
![python-not supported syntax check](https://user-images.githubusercontent.com/25406424/188084319-90a8567c-021b-464d-a803-1e7d30e942aa.png)

### And Code Formatting also not supported it seems as below for python-mode.
![python-not supported code formatter](https://user-images.githubusercontent.com/25406424/188085201-24f563d5-4f22-4245-8455-db70cc9346ca.png)

**Pls help to enable the 'Code Formatter' and 'Syntax Validation' for 'Python' mode**

**Comment by nightwing** [comment]
There is no existing linter for python written in javascript. It may be possible to compile one of existing language servers into javascript using pyscript, or use the parser from [skulpt](https://skulpt.org/), but that seems to be a large task.
